db_schema/schema.sql

```diff
@@ -524,3 +524,69 @@ INSERT INTO appleCountries (id, name, code, ccc) VALUES (4, "Afghanistan", "AF",
 (887, "Yemen", "YE", "yem"),
 (894, "Zambia", "ZM", "zmb"),
 (716, "Zimbabwe", "ZW", "zwe");
+
+CREATE TABLE IF NOT EXISTS anchorEpisodeMetadata (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  ep_name VARCHAR(2048) NOT NULL,
+  ep_url VARCHAR(2048),
+  ep_artwork_url VARCHAR(2048),
+  ep_release_date DATE,
+  ep_description TEXT,
+  ep_explicit BOOLEAN,
+  ep_duration INTEGER,
+  ep_language VARCHAR(100),
+  PRIMARY KEY (account_id, episode_id)
+);
+
+CREATE TABLE IF NOT EXISTS anchorEpisodeDetails (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  aed_date DATE NOT NULL,
+  aed_playscount INTEGER NOT NULL,
+  aed_totaltimelistened BIGINT NOT NULL,
+  aed_uniqueengagedlistenerscount INTEGER NOT NULL,
+  aed_uniquelistenerscount INTEGER NOT NULL,
+  aed_play_histogram JSON,
+  aed_play_top_cities JSON,
+  aed_play_top_countries JSON,
+  PRIMARY KEY (account_id, episode_id, aed_date)
+);
+
+CREATE TABLE IF NOT EXISTS anchorTrendsEpisodeListeners (
```

Comment:
is it really called trends? otherwise I would try to give table names that are as close as possible to the api name